src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -648,6 +648,77 @@
     .power 10
     $.equal-to 2655.992279142402
 
+[] > sqrt-check-1
```

Comment:
@levBagryansky maybe it would be better to stick to naming conventions in other tests? I don't think that name `sqrt-check-1` describes well what is this test about. Maybe it would be better to name it `check-sqrt-zero`?